.gitignore

```diff
@@ -1,3 +1,6 @@
 .project
 .idea
 node_modules
+!intl/en/
+intl/*
+
```

Comment:
>I m not sure who originally decided the three line's standard, but if the original decider requires three, then probably we should keep !intl/.
ATM i think the only difference is if a repo doesn't have folder en, !intl makes sure the intl folder(empty in this case) still exist in that repo

@0candy Tested this in her initial run and @Amir-61  found out you don't need the first line. Hence the update in globalization.md PR. These are nitpicks if you already have those lines in your files anyways. It's up to you since you're the repo owner whether you want to address them now or in the future or if ever. This won't stop your PR from getting the green light.